db/schema.rb

```diff
@@ -18,18 +18,15 @@
   create_table "cars", force: :cascade do |t|
     t.string "manufacturer"
     t.string "model"
-    t.integer "production_year"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "user_id"
+    t.bigint "user_id"
```

Comment:
Is there any real need to change `user_id` type to `bigint`?

Reply:
generated on `bundle install`, probably `pg 1.1.3` update had something in common